src/generateNumber.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const NUM_LENGTH = 4;
+
+module.exports.generateNumber = () => {
+  const digitsArray = [];
+
+  while (digitsArray.length < NUM_LENGTH) {
+    const numToPush = Math.floor(Math.random() * 10);
+
+    if (!digitsArray.includes(numToPush)) {
+      digitsArray.push(numToPush);
+    }
+  }
+
+  return digitsArray.join('');
+};
```

Comment:
you func cant create number with duplicate number and can create number that start from 0, that not so logic